site/source/tutorials/000_local_installation/index.md

```diff
@@ -17,30 +17,41 @@ First, we need to create a Dojo 2 project. Dojo 2 is powerful and leverages adva
 npm install -g @dojo/cli
 ```
 
-This command installs the Dojo command-line tool (`@dojo/cli`) to simplify development tasks related to creating, testing, and building your applications. Initially, the `@dojo/cli` tool comes with three commands:
-* `create` - Provides a skeleton template for Dojo 2 projects
+This command installs the Dojo command-line tool (`@dojo/cli`) to simplify development tasks related to creating, testing, and building your applications. Initially, the `@dojo/cli` tool only comes with two options:
 * `eject` - Disconnects a project from `@dojo/cli` allowing advanced users to customize configuration
 * `version` - Reports the version of `@dojo/cli` and any commands that are installed
 
-In order to create your first Dojo 2 application, run the following command in the directory that will host the new application:
+Running `dojo` will show all available commands even if they aren't installed, when you try to run a command that is not available the CLI will print out a message saying that the command needs to be installed.
```

Comment:
print out a message saying that the command needs to be installed -> print a message that the command needs to be installed